app/src/main/java/ca/bc/gov/vaxcheck/ui/scanner/BarcodeScannerFragment.kt

```diff
@@ -100,7 +101,13 @@ class BarcodeScannerFragment : Fragment(R.layout.fragment_barcode_scanner), Scan
         }
 
         binding.txtPrivacyPolicy.setSpannableLink {
-           findNavController().navigate(R.id.action_barcodeScannerFragment_to_webViewFragment)
+            val webpage: Uri = Uri.parse(getString(R.string.url_privacy_policy))
+            val intent = Intent(Intent.ACTION_VIEW, webpage)
+            try {
+                startActivity(intent)
+            } catch (e: Exception) {
+                Toast.makeText(context, getString(R.string.no_app_found), Toast.LENGTH_LONG).show()
```

Comment:
As displaying a Toast will be something common, I would recommend:
- [ ] creating an extension function from **Context** or: **Activity** and  **Fragment** receiving a text that will display a Long or a Short Toast
- [ ] Call the extension function **here** and on **OnBoardingFragment**  as well